prowler/providers/aws/services/sns/sns_topics_no_http_subscriptions/sns_topics_no_http_subscriptions.py

```diff
@@ -0,0 +1,30 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.sns.sns_client import sns_client
+
+
+class sns_topics_no_http_subscriptions(Check):
+    def execute(self):
+        findings = []
+        for topic in sns_client.topics:
+            for subscription in topic.subscriptions:
+                if subscription.SubscriptionArn == "PendingConfirmation":
+                    continue
+                report = Check_Report_AWS(self.metadata())
+                report.region = topic.region
+                report.resource_id = topic.name
+                report.resource_arn = topic.arn
+                report.resource_tags = topic.tags
```

Comment:
What is the resource checked within the check? If it is the subscription, we need to modify the check name to something like `sns_subscriptions_not_using_http_endpoints` or similar and set the `subscription` as `resource_id` and `resource_arn`.